src/data_template_generators.js

```diff
@@ -63,6 +54,26 @@ const generateDataTemplate = (courseData, pathLookup) => {
 
 const generateLegacyDataTemplate = (courseData, pathLookup) => {
   const dataTemplate = generateDataTemplate(courseData, pathLookup)
+  dataTemplate["publishdate"] = courseData["first_published_to_production"]
+    ? moment(
+      courseData["first_published_to_production"],
+      INPUT_COURSE_DATE_FORMAT
+    ).format()
+    : ""
+  delete dataTemplate["department_numbers"]
+  dataTemplate["departments"] = helpers.getDepartments(courseData)
+  dataTemplate["level"] = {
+    level: courseData["course_level"],
+    url:   helpers.makeCourseInfoUrl(courseData["course_level"], "level")
+  }
+  dataTemplate["course_features"] = courseData["course_feature_tags"]
```

Comment:
Can we do `courseData["course_feature_tags"] || []` to avoid the ternary operator?